lms/envs/common.py

```diff
@@ -2160,6 +2160,9 @@
     # Customized celery tasks, including persisting failed tasks so they can
     # be retried
     'openedx.core.djangoapps.celery_utils',
+
+    # Ability to detect and special-case crawler behavior
+    'openedx.core.djangoapps.crawlers',
```

Comment:
I was debating whether this should be named more generically and not specific to crawlers.  However, this is fine for now.